src/network.zig

```diff
@@ -438,7 +438,7 @@ pub const ConnectionManager = struct { // MARK: ConnectionManager
 
 	pub fn deinit(self: *ConnectionManager) void {
 		for(self.connections.items) |conn| {
-			conn.disconnect();
+			conn.disconnect(.expectedClose, "Server stopped.");
```

Comment:
Does that mean it prints a message on an expected close?
That doesn't seem right.

Reply:
Oh right that makes sense. I was thinking this message would show up in singleplayer, when closing the server.
In that case it would be better to have a separate enum value, instead of sending arbitrary strings (this would make sense for user-provided kick messages, but we don't have those, and it's not yet clear if we want to have them).